plume-models/src/blogs.rs

```diff
@@ -195,6 +195,23 @@ impl Blog {
         }
     }
 
+    pub fn find_by_host(c: &PlumeRocket, host: Host) -> Result<Blog> {
+        let from_db = blogs::table
+            .filter(blogs::custom_domain.eq(host))
+            .limit(1)
+            .load::<Blog>(&*c.conn)?
+            .into_iter()
+            .next();
+        if let Some(from_db) = from_db {
+            Ok(from_db)
+        } else {
+            // we should never get here, because
```

Comment:
(this is actually false, load can return a success with zero elements (type `select * from blogs where custom_domain = 'none';` in an sql shell if you need to convince yourself, no error, but nothing matched))

Reply:
i understand, but this function is only called after our `guard` already put us here

Reply:
removed comment.